### PR TITLE
[ZT] WARP common issues

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/connections/connect-devices/warp/troubleshooting/common-issues.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+<div class="medium-img">
+
+![Example of WARP client UI when unable to turn on WARP](/cloudflare-one/static/documentation/connections/warp-stuck-on-disconnected.png)
+
+</div>
+
+In your [WARP debug logs](/cloudflare-one/connections/connect-devices/warp/troubleshooting/warp-logs), `daemon.log` will typically show one or more of the following errors:
+
+- Happy Eyeball checks failing:
+    ```txt
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/troubleshooting/common-issues.md:24]: @kkrum replace with up-to-date example

File: content/cloudflare-one/connections/connect-devices/warp/troubleshooting/common-issues.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+#### Solution
+
+Configure the third-party service to exempt the [IP addresses required by WARP](/cloudflare-one/connections/connect-devices/warp/deployment/firewall/).
+
+### A third-party VPN is interfering with WARP
+
+The most common places we see interference with WARP from VPNs are:
+
+- **Control of traffic routing:** In `daemon.log`, you will see a large number of routing table changes. For example,
+    ```txt
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/troubleshooting/common-issues.md:48]: @kkrum replace with up-to-date example

File: content/cloudflare-one/connections/connect-devices/warp/troubleshooting/common-issues.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+    [2021-08-12T15:38:53Z DEBUG warp::warp_service] Routes changed:
+        Added; Interface: 8; Destination: 10.133.27.203/32; Next hop: 100.64.0.2; 
+        Added; Interface: 8; Destination: 10.133.27.204/32; Next hop: 100.64.0.2; 
+    [2021-08-12T15:38:54Z DEBUG warp::warp_service] Routes changed:
+        Added; Interface: 8; Destination: 10.133.27.205/32; Next hop: 100.64.0.2;
+    ```
+    This indicates that a third-party VPN is fighting WARP for control over the routing table.
+- **Control of DNS:** In `daemon.log`, you will see a large number of DNS changes followed by a `...<insert log message>..` warning. For example,
+    ```txt
+    [<insert fight for DNS control>]
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/troubleshooting/common-issues.md]: @kkrum add example

File: content/cloudflare-one/connections/connect-devices/warp/troubleshooting/common-issues.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+        Added; Interface: 8; Destination: 10.133.27.201/32; Next hop: 100.64.0.2; 
+        Added; Interface: 8; Destination: 10.133.27.202/32; Next hop: 100.64.0.2; 
+    [2021-08-12T15:38:53Z DEBUG warp::warp_service] Routes changed:
+        Added; Interface: 8; Destination: 10.133.27.203/32; Next hop: 100.64.0.2; 
+        Added; Interface: 8; Destination: 10.133.27.204/32; Next hop: 100.64.0.2; 
+    [2021-08-12T15:38:54Z DEBUG warp::warp_service] Routes changed:
+        Added; Interface: 8; Destination: 10.133.27.205/32; Next hop: 100.64.0.2;
+    ```
+    This indicates that a third-party VPN is fighting WARP for control over the routing table.
+- **Control of DNS:** In `daemon.log`, you will see a large number of DNS changes followed by a `...<insert log message>..` warning. For example,
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/troubleshooting/common-issues.md]: To do: fill in warning message once we have the example

**Review by kkrum — APPROVED** [review]

**Pull request merged**